rls-quick-start-guide.md

```diff
@@ -36,8 +44,6 @@ Rands Leadership Slack has many (many) channels. We could not hope to describe a
 - **`#help-and-advice`:** If there's not a more specific channel that's relevant to your question, you can always ask in this channel. 
 - **`#anonymous-help-and-advice`:** For various reasons, there are times where someone may not feel comfortable asking for advice publicly in RLS. This channel was created to support those times and allow members to create an anonymous top-level post using a channel workflow. The donwside is that it can remove the ability to have deeper discussions, since the original poster cannot respond anonymously in threads to participate.
 - **`#lobby-*`:** Lobby channels are public channels that enable members to request admission to private channels, which would otherwise be undiscoverable. Right now only two exist, `#lobby-adoption` and `#lobby-non-binary-and-women`, and you can see past discussions about the implementation and use of `#lobby-*` channels in `#rands-slack-culture`.
-- **`🔒#the-treehouse`:** a private (but not secret) channel for non-binary folks and women. If you'd like to join this channel, please use the workflow in `#lobby-non-binary-and-women` to make a request!
```

Comment:
I’m a little surprised about removing this. Yes, there are other private channels for under-represented groups, but I’d suspect #the-treehouse is the biggest, and possibly has been around the longest. Including a public mention of it here certainly seemed like a good idea to me, and you did run it by there and get some thumbs up too!

Reply:
@marilyn One of my thoughts was around singling out this channel for notice but not including other channels for under-represented or historically excluded groups. 

@rands Any thoughts on this?

Reply:
(accidentally commented under my work account 🤦🏼 deleted and re-added my comment, sorry for the double pings)

@marilyn One of my thoughts was around singling out this channel for notice but not including other channels for under-represented or historically excluded groups.

@rands  Any thoughts on this?

Reply:
I know there's a private channel for people of color, I could check whether they'd want to be included in this list too. These are super valuable imo and not super easy to find, so I love the idea of calling them out somehow.

Reply:
Agreed on both counts! I think that would be really beneficial to new folks - yes please for checking whether they'd like to be included (and related but outside the scope of this PR, if they'd want to set up a lobby channel).

Reply:
I've added treehouse back in, as well as two other channels; waiting to hear back on a couple others. Especially with RLS being such a big place, I like having more of these community-within-community channels for people.

Reply:
@marilyn Thoughts on the three channels being mentioned here?

Reply:
Love it @angelariggs!!

Reply:
Awesome! I'm going to go ahead and merge this PR since it's been open for a month with no critical feedback :)